python/py-ansible/Portfile

```diff
@@ -5,7 +5,7 @@ PortGroup           python 1.0
 PortGroup           select 1.0
 
 name                py-ansible
-version             2.9.10
+version             2.9.17
```

Comment:
why not updating to the latest version (2.10.7)?

Reply:
2.10.x has some major changes, and just updating version didn't work on my machine. Unfortunately I'm not python expert enough to resolve the issues.

Reply:
looking into it right now

Reply:
Will be closed by https://github.com/macports/macports-ports/pull/10063